Runtime/com.unity.cinemachine.asmdef

```diff
@@ -1,14 +1,17 @@
 {
     "name": "Cinemachine",
+    "rootNamespace": "",
     "references": [
-        "Unity.Timeline",
+        "GUID:f06555f75b070af458a003d92f9efb00",
```

Comment:
Somehow you had "use GUIDS" checked in the asmdef inspector.  We want to use human-readable names, not GUIDS